OpenTreeMap/src/OTM/Controllers/OTMInstanceSelectTableViewController.m

```diff
@@ -70,39 +74,70 @@ - (void)viewWillAppear:(BOOL)animated
         [self setLocationManager:[[OTMLocationManager alloc] initWithDistanceRestriction:NO]];
     }
 
-    // TODO: Maybe show loading indicator
-    [[self locationManager] findLocation:^(CLLocation *location, NSError *error) {
+    [self loadInstances];
+}
+
+- (void)loadInstances
+{
+    if ([[[SharedAppDelegate loginManager] loggedInUser] loggedIn]) {
+        [self.activityIndicatorView startAnimating];
+        [[[OTMEnvironment sharedEnvironment] api]
+         getInstancesNearLatitude:0
+         longitude:0
+         user:[[SharedAppDelegate loginManager] loggedInUser]
+         maxResults:5
```

Comment:
Good catch. That is actually how it works (My user has a dozen or so that all show up). It is confusing that the API ignores this value when searching for the 0,0 locatio. I'll make an overload in the API wrapper to clarify this.